Jenkinsfile

```diff
@@ -46,25 +48,100 @@ pipeline {
             }
         }
 
-        stage('Deploy') {
+        stage('Deploy MHS') {
             steps {
-                dir('pipeline/terraform/test-environment') {
-                    sh label: 'Initialising Terraform', script: 'terraform init -input=false'
+                dir('pipeline/terraform/integration-test-environment') {
+                    sh label: 'Initialising Terraform', script: """
+                            terraform init \
+                            -backend-config="bucket=${TF_STATE_BUCKET}" \
+                            -backend-config="region=${TF_STATE_BUCKET_REGION}" \
+                            -backend-config="dynamodb_table=${TF_MHS_LOCK_TABLE_NAME}" \
+                            -input=false
+                        """
+                    sh label: 'Destroy any existing MHS services', script: """
+                            terraform destroy -auto-approve \
```

Comment:
This command was added as, before it was here, ECS tasks from the previous build would take a while to get destroyed (ECS service would start the tasks for the new build and then get rid of the previous tasks over ~10 minutes), which resulted in the tasks from the previous build handling some requests. However, this command is taking ~6.5 minutes to run ie it slows the build down. We (me and @gareth-allan) believe this is to do with https://github.com/terraform-providers/terraform-provider-aws/issues/3414 ie we think this is a known issue. Nevertheless, with this command, the builds seem to be working correctly.